src/tensor/traits.ts

```diff
@@ -118,6 +118,9 @@ const isPalindrome = (str: string) =>
     .reverse()
     .join('');
 
+const serializeTraitType = (traitType: string) =>
+  traitType.toLowerCase().replace(/[_\s]/g, '');
```

Comment:
Have you tested this on all the domain collections to ensure that this hack works? Any edge cases we're missing (eg plural, no "only" keyword, etc)

Reply:
I did, but I think it's important that I add extra tests to show that with a comment just in case it breaks later on

Reply:
adding rn

Reply:
I'm not sure if its worth matching for plural. None of the collections really do that